src/components/GameCards/GameCard.js

```diff
@@ -35,6 +39,40 @@ export default {
         && this.progresses[this.platform.code][this.game.id];
     },
 
+    releaseDate() {
+      const releaseDate = this.game
+        && this.game.release_dates
+        && this.game.release_dates.filter(({ platform }) => this.platform.id === platform)
+        && this.game.release_dates.filter(({ platform }) => this.platform.id === platform)[0].date;
+
+      const daysUntilRelease = Math
+        .ceil((new Date(releaseDate * 1000) - new Date()) / (1000 * 60 * 60 * 24));
+
+      if (daysUntilRelease > 0) {
+        return daysUntilRelease;
+      } else if (daysUntilRelease === 0) {
+        return 'Today';
+      } else if (daysUntilRelease < 0) {
+        return '';
+      }
+
+      return 'TBA';
+    },
+
+    releaseDateText() {
+      if (this.releaseDate > 1) {
+        return `Releases in ${this.releaseDate} days`;
+      } else if (this.releaseDate === 1) {
+        return 'Releases tomorrow';
+      } else if (this.releaseDate === 'Today' && new Date().getHours() < 15) {
```

Comment:
`new Date().getHours() < 15` what is this for?

Reply:
It's an arbitrary check if it's 3pm. I'm just gonna assume that the game has been released by 3pm. This way it won't show "releases today" all day long, but "released today" at some point.